sources/iTermHyperlinkTrigger.m

```diff
@@ -0,0 +1,69 @@
+//
+//  iTermHyperlinkTrigger.m
+//  iTerm2
+//
+//  Created by leppich on 09.05.18.
+//
+
+#import "PTYScrollView.h"
+#import "PTYSession.h"
+#import "VT100Screen.h"
+#import "iTermURLMark.h"
+#import "iTermURLStore.h"
+#import "NSURL+iTerm.h"
+#import "iTermHyperlinkTrigger.h"
+
+#import <CoreServices/CoreServices.h>
+#import <QuartzCore/QuartzCore.h>
+
+@implementation iTermHyperlinkTrigger
+
++ (NSString *)title {
+    return @"Hyperlink";
+}
+
+- (BOOL)takesParameter {
+    return YES;
+}
+
+- (NSString *)paramPlaceholder
+{
+    return @"https://\\0";
```

Comment:
@gnachman for this use case, it would be nice if the `paramPlaceholder` would be used as a default value.

Reply:
added a `paramDefault` method to `Trigger` in 95849ca